src/components/StoreIssueBlock.js

```diff
@@ -16,46 +19,64 @@ class StoreIssueBlock extends Component {
 
   onHashChange(event) {
     this.setState({
-      certificateHash: event.target.value
+      certificateHash: event.target.value,
+      certificateHashIsValid: isValidCertificateHash(event.target.value)
     });
   }
 
   onIssueClick() {
     const { adminAddress, storeAddress, handleCertificateIssue } = this.props;
-    handleCertificateIssue({
-      storeAddress,
-      fromAddress: adminAddress,
-      certificateHash: this.state.certificateHash
-    });
+    const { certificateHash } = this.state;
+    if (isValidCertificateHash(certificateHash)) {
+      handleCertificateIssue({
+        storeAddress,
+        fromAddress: adminAddress,
+        certificateHash
+      });
+    } else {
+      this.setState({
+        certificateHashIsValid: isValidCertificateHash(certificateHash)
+      });
+    }
   }
 
   render() {
+    const { certificateHash, certificateHashIsValid } = this.state;
+    const { issuingCertificate, issuedTx, networkId } = this.props;
+    const certificateHashMessage = certificateHashIsValid
+      ? ""
+      : "Merkle Root Hash is not valid.";
```

Comment:
```suggestion
      : "Merkle root is not valid.";
```

Am thinking to either use "Merkle root" or "Hash".